include/dxc/HLSL/DxilPipelineStateValidation.h

```diff
@@ -71,6 +71,10 @@ struct PSVRuntimeInfo1 : public PSVRuntimeInfo0
 {
   uint8_t ShaderStage;              // PSVShaderKind
   uint8_t UsesViewID;
+  union {
+    uint16_t MaxVertexCount;          // MaxVertexCount for GS only (max 1024)
+    uint8_t SigPatchConstantVectors;  // Output for HS; Input for DS
```

Comment:
Because this is blittable, it would be good to have both union members be 16-bit wide for clarity (i.e., whether the other 8 bits of padding are before or after SigPatchConstantVectors).